test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -21,8 +21,9 @@
 #   author: First Last (@name) [optional anything]
 #     "Ansible Core Team" - Used by the Bot
 #     "Michael DeHaan" - nop
+#     "OpenStack Ansible SIG" - OpenStack does not use GitHub
 #     "Name (!UNKNOWN)" - For the few untraceable authors
-author_line = re.compile(r'^\w.*(\(@([\w-]+)\)|!UNKNOWN)(?![\w.])|^Ansible Core Team$|^Michael DeHaan$')
+author_line = re.compile(r'^\w.*(\(@([\w-]+)\)|!UNKNOWN)(?![\w.])|^Ansible Core Team$|^Michael DeHaan$|^OpenStack Ansible SIG$')
```

Comment:
We shouldn't be adding content-specific rules to the validation code. Why not allow any non `@` prefixed string to be used for collections instead? That will preserve the more strict check for ansible-base while allowing collection authors more flexibility.

Reply:
I'd be more than happy to do that. It doesn't look like there is branching logic for base vs collection in here now, yeah? I'd love a pointer as to how you think that should look.

Reply:
When the schema is constructed we indicate if a collection is being tested:

https://github.com/ansible/ansible/blob/fc448ea54f1595e8d24e72abd9b5245ec6165842/test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py#L327

You should be able to pass `for_collection` to `author` like we already do for `version`:

https://github.com/ansible/ansible/blob/fc448ea54f1595e8d24e72abd9b5245ec6165842/test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py#L335

Reply:
Sweet, thanks! And you think the collection one should basically just be "there should be an author string" Will update.

Reply:
@gundalow @acozine What are your thoughts? Should we keep our current author restrictions for ansible-base and allow any value for author in collections?

Reply:
I was totally working from a stale copy - my bad. The for_collection stuff is clear and obvious - when I'm actually up to date. :)